crates/circuit/src/imports.rs

```diff
@@ -45,6 +45,8 @@ impl ImportOnceCell {
                 .unwrap()
                 .getattr(self.object)
                 .unwrap()
+                .downcast_into()
+                .unwrap()
```

Comment:
Oh, this is totally unnecessary - it was accidentally left over from some playing I did with `ImportOnceCell` that I then (mostly) reverted.

```suggestion
```
```suggestion
                .downcast_into()
                .unwrap()
```